Spigot-Server-Patches/0625-Optimized-tick-ready-check.patch

```diff
@@ -0,0 +1,21 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: lukas <lukasalt98@gmail.com>
+Date: Sun, 27 Dec 2020 17:19:51 +0100
+Subject: [PATCH] Optimized tick ready check
+
+
+diff --git a/src/main/java/net/minecraft/server/World.java b/src/main/java/net/minecraft/server/World.java
+index 9ed21f434c5fb019b74dfe9ee0b802ccc5c07fd8..631a01c6f65c29cf20a3dfcdca29d4de9c8cce92 100644
+--- a/src/main/java/net/minecraft/server/World.java
++++ b/src/main/java/net/minecraft/server/World.java
+@@ -792,8 +792,8 @@ public abstract class World implements GeneratorAccess, AutoCloseable {
+ 
+             if (!tileentity.isRemoved() && tileentity.hasWorld()) {
+                 BlockPosition blockposition = tileentity.getPosition();
+-
+-                if (this.getChunkProvider().a(blockposition) && this.getWorldBorder().a(blockposition)) {
++                
++                Chunk chunk; PlayerChunk playerChunk; if ((chunk = tileentity.getCurrentChunk()) != null && (playerChunk = chunk.playerChunk) != null && playerChunk.isTickingReady() && this.getWorldBorder().a(blockposition)) { // Paper - optimized tick ready check
```

Comment:
Add a comment of what exactly you did (i.e. inlined the `getChunkProvider().a(BlockPosition)` call).

Reply:
Alright, I've added a comment